code/docker-compose.yaml

```diff
@@ -7,6 +7,8 @@ services:
     environment:
       - MYSQL_ALLOW_EMPTY_PASSWORD=true
   etherpad:
-    image: etherpad/etherpad
+    image: etherpad/etherpad:stable
     ports:
-      - "9001:9001"
\ No newline at end of file
+      - "9001:9001"
+    volumes:
```

Comment:
Create a volume section to create a mapping between local files on my computer and container files